lib/url.js

```diff
@@ -919,7 +919,7 @@ Url.prototype.resolveObject = function resolveObject(relative) {
   // If a url ENDs in . or .., then it must get a trailing slash.
   // however, if it ends in anything else non-slashy,
   // then it must NOT get a trailing slash.
-  let last = srcPath.slice(-1)[0];
+  let last = srcPath.at(-1);
   const hasTrailingSlash = (
     ((result.host || relative.host || srcPath.length > 1) &&
     (last === '.' || last === '..')) || last === '');
```

Comment:
Unrelated to the change but am I missing something or there is a bug here?
`let last = srcPath.slice(-1)[0];` can only return a string with 1 character or `undefined`.

Reply:
Ah it's an array. Naming and usage made me think it was a string.